tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,29 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    This method writes a set of images to the current log directory.
+    The written data appears in TensorBoard's 'Images' dashboard.
+
+    This example writes 2 random grayscale images:
+
+    ```
+    writer = tf.summary.create_file_writer('logs')
+    with writer.as_default():
+      image1 = tf.random.normal(shape=[8, 8])
+      image2 = tf.random.normal(shape=[8, 8])
+      tf.summary.image("grayscale noise", [image1, image2], step=0)
+    ```
+
+    When image data is outside of the range [0, 1), please rescale the data
```

Comment:
Consider rephrasing to describe a requirement, rather than make a request:

"Data should be scaled to be within the [0, 1) range."

Reply:
Done.